main.js

```diff
@@ -449,7 +450,26 @@
                     fetchWPRDCData(dataSourceName);
                 });
                 retryDiv.appendChild(retryButton);
-            }));
+            }))
+            .then(() => {
+                return fetch(WPRDC_META_URL + dataSource.id)
+                .then((response) => {
+                    //check response for data and data for date
+                    if (response.status >= 200 && response.status < 300) {
+                        return response.json();
+                    }
+                })
+                .then((metadata) => {
+                    // Parse json for last modified field
+                    var parsedDate = Date.parse(metadata.result.last_modified);
+
+                    // Display a notification if the dataset has been updated within the hour
+                    var diff = Date.now() - parsedDate;
+                    if (diff <= 3600000) {
```

Comment:
I wonder if each dataset should designated what "new" is. The 311 dataset is hourly, or however close it gets, but some others are daily or monthly. What do you think of designating a `.newDataNotificationTime`, or something in each of the datasets as a property? Is there enough difference to make sense, do you think?